docs/090-breaking-changes.rst

```diff
@@ -16,6 +16,7 @@ New Restrictions
 ================
 
 - `error` is now a keyword and cannot be used as identifier anymore.
+- Make basefee a reserved identifier in assembly.
```

Comment:
```suggestion
- ``error`` is now a keyword and cannot be used as identifier anymore.
- Make ``basefee`` a reserved identifier in assembly.
```

Reply:
@v-sreekesh Looks like the changes you made here have now disappeared.

I saw you tried to merge `develop` into your branch. This is not necessary. We have solved the problem in #11840 and once we merge it, you'll be able to rebase your PR on the updated `breaking` branch.

Reply:
Ok